src/Form/FieldType/FieldTypeType.php

```diff
@@ -111,7 +112,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]);
         }
 
-        if (null !== $fieldType && null != $fieldType->getChildren() && $fieldType->getChildren()->count() > 0) {
+        if (null != $fieldType->getChildren() && $fieldType->getChildren()->count() > 0) {
```

Comment:
null !=**=** $fieldType->getChildren() ?

Reply:
The null !== has been removed, because $fieldType->getChildren will ALWAYS return a doctrine collection

Reply:
https://github.com/ems-project/EMSCoreBundle/pull/1127/commits/904e4fa42e6cc6ea1f91ea73ff26221d821212fe